lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -275,15 +274,10 @@ sub new_privileged {
     );
 }
 
-sub gravatar {
+sub avatar {
```

Comment:
Maybe add a # TODO comment here?

Reply:
The ticket for “returning avatars from Discourse” is MBS-9725.
You might want to create a separate ticket to refer to the removal of Gravatar in this pull request.

Reply:
(Thanks for opening the ticket, I meant to do that sooner.)